### PR TITLE
docs: explain how to override msbuild location for vscode/winui builds

### PR DESCRIPTION
**Pull request by spouliot**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Before 0.14 it was not possible to override which `msbuild.exe` would be used to build WinUI apps from VSCode.

## What is the new behavior?

With 0.14 it's now possible to override `msbuild.exe` location and this PR documents how to achieve that.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by spouliot** [comment]
note: should only be merged after the Uno extension for VS Code version 0.14 is released.

File: doc/articles/uno-build-error-codes.md [hunk 1]
```diff
@@ -137,3 +137,11 @@ Building for the specified target framework is not supported on the current plat
 Building WinUI applications requires the use of `msbuild` and the extension must be able to find it. This is done by using the `vswhere` utility.
 
 Installing the latest stable Visual Studio release should provide both tools.
+
+With version 0.14 (and later) you can override the location of `msbuild.exe` by
```
Comment (agneszitte) [inline doc/articles/uno-build-error-codes.md:141]:
```suggestion
With version 0.14 (and later) you can override the location of `msbuild.exe` by:
```

File: doc/articles/uno-build-error-codes.md [hunk 2]
```diff
@@ -137,3 +137,11 @@ Building for the specified target framework is not supported on the current plat
...
+
+- opening the **Settings** using `Ctrl`+`,`
+- searching for `msbuild`
+- setting the location where your `msbuild.exe` is located
```
Comment (agneszitte) [inline doc/articles/uno-build-error-codes.md:145]:
```suggestion
1. Opening the **Settings** using `Ctrl`+`,`
1. Searching for `msbuild`
1. Setting the location where your `msbuild.exe` is located
```

**Review by Youssef1313 — APPROVED** [review]

**Review by ajpinedam — APPROVED** [review]
Suggestion from Agnes, but everything else looks good

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**